cmd/subnetcmd/create.go

```diff
@@ -378,10 +379,7 @@ func createSubnetConfig(cmd *cobra.Command, args []string) error {
 				if err != nil {
 					return err
 				}
-			case explainOption:
-				ux.Logger.PrintToUser("The difference is...")
-				ux.Logger.PrintToUser("")
-				continue
+				//missing case for dontChangeFeeSettingsOption
```

Comment:
there is a missing case for the `dontChangeFeeSettingOption`

Reply:
yup. working on the implementation. thanks for this!